tools/pylampo-client/pyproject.toml

```
@@ -0,0 +1,13 @@
+[tool.poetry]
+name = "pylampo-client"
+version = "0.0.1"
+description = "A simple Lampo client that communicates via a Unix socket"
+authors = ["Tarek <tareknaser360@gmail.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

```suggestion
python = "^3.7"
```

I do not see any reason to use 3.8 but not support 3.7. There is any that I am missing?

---

Nothing really. this was auto-generated with `poetry`. 
I can make it `3.7`

---

yeah I was assuming that poetry took your default :)